infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -11,6 +11,7 @@
 
 import java.lang.invoke.MethodHandles;
 import java.util.Map;
+import java.util.Random;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Random'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)